test/csuite/timestamp_abort/main.c

```diff
@@ -80,7 +80,7 @@ static const char *const uri_shadow = "shadow";
 static const char *const ckpt_file = "checkpoint_done";
 
 static bool columns, compat, inmem, stress, use_ts;
-static volatile uint64_t global_ts = 1;
+static volatile uint64_t global_ts = 2;
```

Comment:
This change makes me nervous. It means we have changed when `0` can be used as a timestamp in the API. Was the failure obvious? i.e: are we confident that MongoDB isn't going to need something similar?

Reply:
Yes, it's a loud failure with an obvious error message. I don't think MongoDB will need a similar change since it's not a failure I've seen in my patch builds.